### ISSUE TITLE
Can't seem to get the recipes to load correctly.

### ISSUE THREAD

**Issue by NullMagic** [CyclopsMC/IntegratedDynamics#108]
Alright, to start, I did give this the "good college try". This of course doesn't mean I'm smart enough to have actually done everything right, but I did my best to make it all work on my own.

Alright, so, to begin, it was not clear when I installed this mod that it does not add it's own recipes for all it's items (I might suggest something similar to Ender IO where it automatically installs all the appropriate config files upon launch) so it did take me a bit to figure out I was supposed to go download the recipe xmls, so you may want to consider a bit of clarification in the Curse Description.

After downloading all the files I found that the Integrateddynamics folder I was supposed to put them into never appeared in my config folder (as I'd run the game multiple times since installing the mod, I can only assume that something, somewhere, went wrong). This was simple enough to fix, I just added the folder manually and dropped all the files inside. I then proceeded to play minecraft, thinking all would be well, unfortunately it was not. Despite the appropriate location (which I am fairly certain is correct due to the crash CyclopsCore gave me due to a misplaced space in one of the xmls) the recipes don't seem to do anything.

I tried several things to make them work, including editing the recipes, checking the config to make sure all the correct flags were set, and even creating a _override folder and dropping the files in there. The first two didn't work as changes didn't seem to effect at all what recipes showed and what didn't, and the final had some very alarming results. After placing all the xmls in the _override folder all the recipes which had previously worked and displayed in JEI (such as mehnril chunks to blocks, blocks to bricks, wood to planks, etc) suddenly stopped working. Neither displaying in JEI or functioning in the crafting table, despite the fact that the recipes provided in the xml files were exactly the same as originally displayed without the xml files present.

I am, distressingly, forced to conclude that the recipe xmls don't work for some reason. However, as no one else has reported this, I am increasingly concerned that the problem may be entirely on my end, which makes me wonder what on earth I'm doing wrong, and how the hell I can hope to fix it.

For reference, I am using the 1.11.2 0.6.5 version of the mod. Given that this is the initial port to 1.11 I concede the possibility that it might be the case that it just doesn't work right (yet), and if that's the case I hope it gets fixed soon since this seems to be quite an interesting mod, anyway, sorry for bothering you, please let me know if you have any ideas as to what my problem may be, or if this is a known issue and you're working on it, etc.

**Comment by rubensworks** [comment]
Thanks for reporting this, but could you summarize in a few bullet points what isn't working?
I have a lot of issues like these, and I'd like to be able to quickly skim over them.

**Comment by NullMagic** [comment]
- No folder for Integrated Dynamics is being created in the config folder

- No recipes are automatically added for Integrated Dynamics (possibly due to the missing folder, possibly by design, still not clear on that point).

- Manually added XMLs from your wiki simply do not function (the game is loading them, it crashes if they aren't correctly configured as an XML, they just don't do anything, if added to an _override folder they actively stop existing recipes from functioning)

**Comment by rubensworks** [comment]
- The folder not being created is intended automatically, I've updated the wiki to reflect this.
- Some recipes (some were) were indeed not being loaded, this has been fixed.
- I will look into testing recipe overriding tomorrow.

**Comment by rubensworks** [comment]
@NullMagic I just tried the recipe overriding myself, and everything seems to work as intended.

I've used the following recipes file: https://cl.ly/2n3O020w3B0C
Could you try using that one yourself by placing it on the following location?: `config/integrateddynamics/_override/squeezer_ores.xml`
You should have the recipe available to squeeze one smoothstone into 4 cobblestone.

**Comment by NullMagic** [comment]
I added the xml to the override folder, it didn't seem to affect the squeezer recipes at all, squeezing stone still gives 1 cobble, and all other squeezer recipes are still present.

**Comment by rubensworks** [comment]
@NullMagic I just tried it myself again, even in a non-dev environment, and everything works as intended. So I'm closing this issue.

Make sure to add it like this:
<img width="783" alt="schermafbeelding 2017-01-17 om 20 17 32" src="https://cloud.githubusercontent.com/assets/440384/22035965/11e74c80-dcf2-11e6-856e-744b71c7396e.png">
